### ISSUE TITLE
API Keys

### ISSUE THREAD

**Issue by moqri** [dssg/innovation-ecosystems#6]
We have 3 private apis in the code. How can we remove them and any track of them from old versions?

**Comment by reidjohnson** [comment]
I don't believe it's possible to selectively remove revision history from GitHub (i.e., even if removed now, the API keys would be visible in the version history), so I'm not sure offhand. However, I think it should at least be possible to make the current project private and transfer the relevant content to a public repository.

**Comment by hunterowens** [comment]
This may be of use: https://help.github.com/articles/remove-sensitive-data/
